Grinnell-DB-iOS/ResultsViewController.m

```diff
@@ -19,282 +19,281 @@ @implementation ResultsViewController
+    classLbl.text = year;
+    statusLbl.text = status;
+  } else if ([status isEqualToString:@"Faculty / Staff"]) {
+    NSString *dept = [tempPerson.attributeVals objectAtIndex:[tempPerson.attributes indexOfObject:@"Department"]];
+
+    NSMutableArray *titleArray = [[NSMutableArray alloc] init];
+    NSUInteger index = [tempPerson.attributes indexOfObject:@"Title"];
+    while ([@"Title" isEqualToString:[tempPerson.attributes objectAtIndex:index]]) {
+      NSString *title = [tempPerson.attributeVals objectAtIndex:index];
+      [titleArray addObject:title];
+      index++;
+    }
+    NSString *compositeTitle = [titleArray objectAtIndex:0];
+
+    for (int i = 1; i < titleArray.count; i++) {
+      compositeTitle = [compositeTitle stringByAppendingString:[NSString stringWithFormat:@"/%@", [titleArray objectAtIndex:i]]];
     }
-    
-    UITableViewCell *cell = (UITableViewCell*)[tableView dequeueReusableCellWithIdentifier:cellIdentifier];
-	if (!cell) {
-		cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleDefault reuseIdentifier:cellIdentifier];
-	}
-    
-    // Connect the cell's properties
-    UILabel *nameLbl = (UILabel *)[cell viewWithTag:1002];
-    UILabel *statusLbl = (UILabel *)[cell viewWithTag:1003];
-    UILabel *usernameLbl = (UILabel *)[cell viewWithTag:1004];
-    UILabel *majorLbl = (UILabel *)[cell viewWithTag:1005];
-    UILabel *classLbl = (UILabel *)[cell viewWithTag:1006];
-    UIImageView *userImageView = (UIImageView *) [cell viewWithTag:1007];
-    
-    Person *tempPerson = [[Person alloc] init];
-    tempPerson = [searchDetails objectAtIndex:indexPath.row];
-    
-    NSString *first = tempPerson.firstName;
-    NSString *last = tempPerson.lastName;
-    NSString *status = [tempPerson.attributeVals objectAtIndex:[tempPerson.attributes indexOfObject:@"Status"]];
-    NSString *username = [tempPerson.attributeVals objectAtIndex:[tempPerson.attributes indexOfObject:@"Username"]];
-    
-    // Fetch the image
-    NSUInteger index = [tempPerson.attributes indexOfObject:@"picURL"];
+
+    compositeTitle = [compositeTitle stringByTrimmingCharactersInSet:[NSCharacterSet whitespaceAndNewlineCharacterSet]];
+    majorLbl.text = compositeTitle;
+    classLbl.text = dept;
+    statusLbl.text = @"Fac/Staff";
+  } else {
+    NSUInteger index = [tempPerson.attributes indexOfObject:@"Title"];
     if (NSNotFound != index) {
-        NSString *userImageString = [tempPerson.attributeVals objectAtIndex:index];
-        userImageView.contentMode = UIViewContentModeScaleAspectFit;
-        [userImageView sd_setImageWithURL:[NSURL URLWithString:userImageString] placeholderImage:nil];
+      NSString *title = [tempPerson.attributeVals objectAtIndex:index];
+      majorLbl.text = title;
+      classLbl.text = [tempPerson.attributeVals objectAtIndex:[tempPerson.attributes indexOfObject:@"Department"]];
     } else {
-        userImageView.image = nil;
+      majorLbl.text = @"Data unavailable off campus";
```

Comment:
Yes, as the majorLbl only exists on the off-campus .xib. I'm looking forward to streamlining the interface in a future Changelist.